MdeModulePkg/Universal/BdsDxe/BdsEntry.c

```diff
@@ -1151,8 +1153,6 @@ BdsEntry (
   DEBUG ((DEBUG_ERROR, "[Bds] Unable to boot!\n"));
   PlatformBootManagerUnableToBoot ();
 
-  PlatformBootManagerDeadloop ();    // MU_CHANGE

```

Comment:
We can also drop all the PlatformBootManagerDeadloop stuff here, right, since we added it? Maybe do that in a separate commit and mark it squash on rebase and we can get rid of it on the next rebase entirely.

Reply:
Updated that in the 5th commit.